yang-models/dell-base-pas.yang

```diff
@@ -14,18 +14,21 @@ module dell-base-pas {
 
   description "Configuration and operational data for OS10 Platform Abtraction Service";
 
-  revision 2015-04-03 {
-    description "Initial revision.";
+  revision 2018-09-21 {
```

Comment:
revision "2018-08-10" {
        description "Fixing pyang IETF errors and adding documentation to the model.";
        reference "Network Platform Abstraction";
    }

These changes are redundant mostly. The modified files to be synced soon. I would suggest you can wait for these files to get the updated changes, and then re-check for pyang validity. Most of them are taken care of. If any remaining, you can fix them.